src/content/ui-components/components/blockquote-button.mdx

```diff
@@ -53,15 +54,13 @@ export default function MyEditor() {
         <blockquote>
             <p>"The best way to predict the future is to invent it."</p><p>— Alan Kay</p>
         </blockquote>
-        <pre><code>console.log('Hello, World!');</code></pre>
```

Comment:
Why delete this?

Reply:
Since the node button is split into two components, the blockquote demo should only contain the blockquote itself, not the code demo.